content/confirmation/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+private: true
+title: Confirmation
+meta_desc: This page is provides confirmation about register for a Pulumi event, webinar, or workshop.
+block_external_search_index: true
+---
```

Comment:
Do we ever expect to use this index page? Instead of having it as effectively empty page, we could consider just redirecting to the homepage, e.g. changing the content of this file to:

```markdown
---
redirect_to: /
---
```

Reply:
Yup, that worked. I removed all the extra stuff and just have the redirect now.